src/compressor.c

```diff
@@ -82,8 +82,17 @@ int main(int argc, char **argv) {
             rate_limit = 12000;
             fprintf(stderr, "Warning: no rate limit set; defaulting to 12000\n");
         }
+
+        int tcp_exclude = 0;
+
+        if (config_lookup_int(&config, "tcp_exclude", &tcp_exclude) == CONFIG_FALSE)
+        {
+            tcp_exclude = 0;
+        }
+
         cfg.rate_limit = rate_limit;
         cfg.new_conn_limit = new_conn_limit;
+        cfg.tcp_exclude = tcp_exclude;
```

Comment:
Where is `tcp_exclude` enabled? It looks like `tcp_exclude` will never be non-zero here.

Reply:
`config_lookup_int()` looks it up inside of the config file. If it doesn't find anything, it sets it to 0 (default). It is also assigned to the config struct variable at line 95 in `src/compressor.c`.

Compressor config example:

```
interface = "ens3";

//redis_cache = {
//    address = "149.28.241.13",
//    //address = "1.1.1.1",
//    port = 6379
//};

// IP-Rate limiting.
ip_rate_limit = 12000;
new_conn_limit = 30;
tcp_exclude = 1;
```

Reply:
Pointers 🤦